cmd/infra/powervs/create.go

```diff
@@ -362,6 +362,9 @@ func (infra *Infra) SetupInfra(options *CreateInfraOptions) error {
 // setupSecrets generate secrets for control plane components
 func (infra *Infra) setupSecrets(options *CreateInfraOptions) error {
 	var err error
+
+	deleteSecrets(options.Name, options.Namespace, infra.AccountID, infra.ResourceGroupID)
```

Comment:
what if these secrets are used by any other existing cluster?

Reply:
That is possible only if we try to create duplicate cluster with same name.
In same management cluster, it fails before infra creation process.
If its diff management cluster, then this could be an issue, but how can we validate that its belongs to different cluster from an unknown management cluster.

Reply:
I don't see a way to avoid this, only way is to prompt user about this error and may be add an additional flag to forcefully recreate these secrets like `--recreate-creds`

Reply:
Yes that's a good idea, will add this flag and delete the secrets only if its enabled.

Reply:
Added a flag and refactored to delete only if this flag is enabled. Ptal!